### DISCUSSION TITLE
課題08についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#24] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/pull/23#issue-1300380185

↑上記のリンク先に質問を書かせていただきました。
よろしくお願いいたします。

**Comment by nsc-kiwamu** [comment]
https://github.com/nsc-kiwamu/traning-takeda/pull/23#issuecomment-1180446398
